src/entity/Blog.ts

```diff
@@ -0,0 +1,41 @@
+import {Entity, PrimaryGeneratedColumn, Column, ManyToOne, OneToMany} from "typeorm";
+
+enum BlogStatus {
+  private = 'private',
+  public = 'public'
+}
+
+import {Category} from "./Category"; 
+import {Like} from "./Like"; 
+import {BlogImage} from "./BlogImage"; 
+
+@Entity()
+export class Blog {
+  
+  @PrimaryGeneratedColumn('increment')
+  id: number;
+  
+  @Column()
+  title: string;
+  
+  @Column()
+  content: string;
+  
+  @Column()
+  main_img: string;
+  
+  @Column('text')
+  status: BlogStatus;
+  
+  @Column({ type: 'timestamp', default: () => 'CURRENT_TIMESTAMP'})
+  created_at: string;
+  
+  @ManyToOne(() => Category, (category) => category.blogs)
+  category: Category
+
+  @OneToMany(() => Like, like => like.blog)
+  likes: Like[];
+
+  @OneToMany(() => BlogImage, blogImage => blogImage.blog)
+  blogImage: BlogImage[];
```

Comment:
why some column names are snake case, some of them are camel case. What I have suggested to you ?
`@Column({ name: "first_name", type: "varchar" })
    public readonly firstName!: string;`

Snake case version is database column, camel case version is for validation

Reply:
Change all entities. Pay close attention to naming